Updated consistent use of '$' (#98)
cedadev/jasmin-workshop: exercises/ex00/README.md

<details>
  <summary>Mac or Linux instructions</summary>

>   In the Mac or Linux environments, the `$` symbol is used to indicate the start of a terminal promt. It is in the example lines to help you practice writing out the commands rather than copying/pasting them.

Reviewer: ```suggestion
  In the Mac or Linux environments, the `$` symbol is used to indicate the start of a terminal prompt. We have put them in the example commands to help you practice writing them out rather than copying/pasting them.
```
